tribute-page/js/index.js

```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
```

Comment:
You should remove this index.js file as there is no code inside a file, there are only comments so you can delete this file

Reply:
- [ ] Kindly make sure you rephrase your comments and make them in a friendly and polite way.
- [ ] Also, always explain why the changes are suggested :smile: